source/manual/fall-back-to-mirror.html.md

```diff
@@ -1,125 +1,113 @@
-`govuk-<environment>-mirror`. This primary bucket is automatically replicated to another S3 bucket
-(named `govuk-<environment>-mirror-replica`) in another region by AWS.
+Then the [`govuk_crawler_worker`](https://github.com/alphagov/govuk_crawler_worker) on the mirrorer machine:
 
-In addition, the primary AWS S3 bucket `govuk-<environment>-mirror` is synced to the Google GCS bucket
-of the same name daily at 12:00.
+- consumes the fetched GOV.UK URLs from the RabbitMQ message queue
+- retrieves the GOV.UK HTML files returned by these URLs
+- saves these HTML files to a `/tmp` folder on the mirrorer machine
+- adds any new URLs found on those pages to the back of the RabbitMQ message queue
 
-The crawler is entirely independent of the mirrors. Stopping the crawler means no new updates are made
-to the mirrors, but it will not stop the mirrors from working.
+Every hour, the [`govuk_sync_mirror`](https://github.com/alphagov/govuk-puppet/blob/86d1480c6e081313c415246063d5931af24473da/modules/govuk_crawler/manifests/init.pp#L109) script runs on the mirrorer machine. This script copies the GOV.UK HTML files from the mirrorer machine to the main `govuk-<environment>-mirror` AWS S3 bucket. AWS then copies this main bucket to the replica `govuk-<environment>-mirror-replica` S3 bucket in another region.
 
-To inspect the contents of the mirror:
+Finally, a job is run in Google Cloud Storage (GCS) at 6:00pm the day after the original `govuk_seed_crawler`script is run. This job syncs the primary AWS S3 bucket `govuk-<environment>-mirror` to the GCS `govuk-<environment>-mirror` bucket. For more information on GCS, see the [Google Cloud Platform documentation](https://docs.publishing.service.gov.uk/manual/google-cloud-platform-gcp.html).
 
-```bash
+The `govuk_seed_crawler` and `govuk_crawler_worker` scripts are independent of the mirrors. Stopping these scripts stops the mirror updates, but does not stop the mirrors from working.
+
+Run the following to inspect the contents of the GOV.UK mirrorer machine:
+
+```
 gds govuk connect -e production ssh mirrorer
 cd /mnt/crawler_worker/www.gov.uk
 ```
 
+## `Mirror GOV.UK content to S3` alert
+
+If the `govuk_sync_mirror` cronjob has not succeeded for 24 hours, it triggers the ‘Mirror GOV.UK content to S3’ alert. See the [Mirror GOV.UK content to S3 alert documentation](/manual/alerts/mirror-sync.html) for more information.
+
 ## Forcing failover to the static mirrors
 
-Because the CDN will retry every request against the mirrors automatically if origin is unavailable,
-[stopping Nginx on the cache machines with Fabric][fab-fail] will result in falling back to mirrors:
+If Origin is unavailable, Fastly will automatically retry every request against the mirrors. This means stopping [Nginx](https://www.nginx.com/) on the cache machines will result in user requests going to the mirror. To stop Nginx on the cache machines, run the following command:
```

Comment:
Hmmm I've just noticed that there are two approaches to this in the dev docs - the one here, and the one described in https://docs.publishing.service.gov.uk/manual/fall-back-to-aws-cloudfront.html. I.e. switching to AWS CloudFront gives us the "failover to the static mirrors" behaviour, although it's a blunter tool than disabling Nginx on specific machines.

Given the fabric scripts are deprecated, switching to CloudFront is probably the preferred approach here. That said, that's only because AWS CloudFront currently only serves static content; there's nothing stopping us having AWS CloudFront making requests to GOV.UK Origin in future, for example. So this might only be a short term best practice:

```suggestion
If Origin is unavailable, Fastly will automatically retry every request against the mirrors. To avoid Fastly traffic hitting Origin while its down (potentially exacerbating its problems), we can [fall back to AWS CloudFront](/manual/fall-back-to-aws-cloudfront.html), which serves all content via the static mirrors.

Alternatively, we can stop [Nginx](https://www.nginx.com/) on the cache machines, which will prevent requests hitting GOV.UK applications. Fastly will automatically retry these failed requests against the mirror.

To stop Nginx on the cache machines, run the following command:
```